website/content/en/status/report-2023-07-2023-09/desktop-convenience.adoc

```diff
@@ -0,0 +1,26 @@
+=== Filling gaps in the FreeBSD desktop experience
+
+Links: +
+link:https://github.com/outpaddling/qmediamanager[External media manager] URL: link:https://github.com/outpaddling/qmediamanager[] +
+link:https://github.com/outpaddling/devd-mount[External media manager] URL: link:https://github.com/outpaddling/devd-mount[] +
+link:https://github.com/outpaddling/npmount[External media manager] URL: link:https://github.com/outpaddling/npmount[] +
+link:https://github.com/outpaddling/freebsd-update-notify[External media manager] URL: link:https://github.com/outpaddling/freebsd-update-notify[] +
+
+Contact: Jason Bacon <jwb@FreeBSD.org>
+
+The sysutils/desktop-installer port, available for over a decade now, quickly configures a bare FreeBSD system with any desktop environment or window manager.
+However, the FreeBSD base and ports collection has been missing some common features that end-users have come to expect from a desktop OS.
+
+The desktop-installer battery monitor script has been enhanced to display popup notifications at various levels of charge/discharge.
+
+The new port, deskutils/qmediamanager, in conjunction with sysutils/devd-mount and sysutils/npmount, displays a popup window when an external storage device is attached, offering the user options to open a file manager, format, or unmount.
+It provides a convenient and secure way to work with external media such as USB sticks.
+
+The sysutils/freebsd-update-notify port displays a popup when new base updates are available, or when a configurable time limit has elapsed.
```

Comment:
```suggestion
The pkg:sysutils/freebsd-update-notify[] port displays a popup when new base updates are available, or when a configurable time limit has elapsed.
```

Reply:
@outpaddling hi, these suggested changes were edited a few hours ago to properly use the `package:` prefix (not `pkg:`). 

My bad with the first edition.